.run_docker_tests.sh

```diff
@@ -18,8 +18,8 @@ python3 -c 'import sys; print(sys.maxsize)'
 
 # The doctestplus plugin in Astropy doesn't work correctly with pytest>=3.2, so
 # we install an older version here
-pip3.5 "pytest<3.2" pytest-xdist
+easy_install-3.5 "pytest<3.2" pytest-xdist
```

Comment:
oh, and we don't need the pytest version limitation any more. For the other CIs it's still around, but astropy has been already fixed.